pkg/types/validation/installconfig.go

```diff
@@ -1345,3 +1352,17 @@ func validateReleaseArchitecture(controlPlanePool *types.MachinePool, computePoo
 
 	return allErrs
 }
+
+// supportedSingleNodePlatform indicates if the IPI Installer can be used to install SNO on
+// a platform.
+func supportedSingleNodePlatform(bootstrapInPlace bool, p *types.Platform) bool {
+	if p.AWS != nil || p.GCP != nil || p.Azure != nil {
```

Comment:
It _appears_ to me that we can add powervs (but not ibmcloud) to this list.
https://issues.redhat.com/browse/OCPBUGS-6508?focusedId=25675347&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25675347